src/components/WheelmapHomeLink.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { t } from 'ttag';
+import styled from 'styled-components';
+
+const WheelmapHomeLink = ({ className, logoURL }) => (
+  <a
+    className={className}
+    href="https://wheelmap.org"
```

Comment:
-1: As you are using the clientSideConfiguration logo, shouldn't this also use the client side configuration hostName and label?

Reply:
This is unclear for me though. Is it the aim of the embed mode to bring "wheelmap" to external pages, or should any white-label app be embeddable?

Reply:
I totally agree. I'm just not sure where to get the "home url" from. It seems like we don't provide that in the `clientSideConfiguration` yet. Let me check

Reply:
Fixed